### PR TITLE
Adding percyCSS options for PoA [Snapshot level]

### PR DESCRIPTION
**Pull request by Amit3200**
Adding percyCSS option for PoA.
This will allow to have user controlled stabilisations while taking screenshots.
The property will be applied and once screenshot is taken it will be removed.

### CODE REVIEW

File: packages/webdriver-utils/src/providers/genericProvider.js [hunk 1]
```diff
@@ -53,12 +53,29 @@ export default class GenericProvider {
...
+  async addPercyCSS(userCSS) {
+    const createStyleElement = `const e = document.createElement('style');
+      e.setAttribute('class', 'poa-user-css-injected');
+      e.innerHTML = '${userCSS}';
+      document.head.appendChild(e);`;
+    await this.driver.executeScript({ script: createStyleElement, args: [] });
+  }
+
+  async removePercyCSS() {
+    const removeStyleElement = `const n = document.querySelectorAll('.poa-user-css-injected');
```
Comment (Amit3200) [inline packages/webdriver-utils/src/providers/genericProvider.js:65]: change this to `attribute` or look at the current `percy injected css`

Comment (Amit3200) [reply packages/webdriver-utils/src/providers/genericProvider.js:65]: query using `document.querySelector(‘[data-percy-css]’)`

File: packages/webdriver-utils/src/providers/genericProvider.js [hunk 2]
```diff
@@ -53,12 +53,29 @@ export default class GenericProvider {
     }
   }
 
+  async addPercyCSS(userCSS) {
+    const createStyleElement = `const e = document.createElement('style');
+      e.setAttribute('class', 'poa-user-css-injected');
+      e.innerHTML = '${userCSS}';
+      document.head.appendChild(e);`;
```
Comment (Amit3200) [inline packages/webdriver-utils/src/providers/genericProvider.js:60]: `append at body`

File: packages/webdriver-utils/test/providers/genericProvider.test.js [hunk 3]
```diff
@@ -115,4 +121,30 @@ describe('GenericProvider', () => {
...
+  describe('addPercyCSS', () => {
+    beforeEach(() => {
+      spyOn(Driver.prototype, 'executeScript').and.returnValue(Promise.resolve(true));
+    });
+
+    it('should call executeScript to add style', async () => {
+      genericProvider = new GenericProvider('123', 'http:executorUrl', { platform: 'win' }, {}, 'local-poc-poa', 'staging-poc-poa', {});
+      await genericProvider.createDriver();
+      await genericProvider.addPercyCSS('h1{color:green !important;}');
+      expect(genericProvider.driver.executeScript).toHaveBeenCalledTimes(1);
```
Comment (itsjwala) [inline packages/webdriver-utils/test/providers/genericProvider.test.js:134]: Have expectation to have been called with and pass arguments as well. So in future if this method changed due to package update we fail the tests

Comment (Amit3200) [reply packages/webdriver-utils/test/providers/genericProvider.test.js:134]: This function of `executeScript` has its own test and is being tested that way.

**Review by itsjwala — APPROVED** [review]
lgtm, some tests are failing

**Pull request merged**